modules/metrics/src/main/java/org/apache/ignite/internal/metrics/exporters/PushMetricExporter.java

```diff
@@ -88,6 +90,10 @@ public void reconfigure(ExporterView newVal) {
         }
     }
 
+    protected void reconfigureInternal(ExporterView view) {
```

Comment:
It seems to me, introducing a new method is redundant. For example, `LogPushExporter` can override `reconfigure` as follows:
```
    @Override
    protected void reconfigure(ExporterView view) {
        LogPushExporterView v = (LogPushExporterView) view;
        oneLinePerMetricSource = v.oneLinePerMetricSource();
        enabledMetrics = Arrays.asList(v.enabledMetrics());

        super.reconfigure(view);
    }
```